### PR TITLE
install urdf rbm, fix compile error, default joint limits for urdf

### PR DESCRIPTION
**Pull request by RussTedrake**
pod install URDFRigidBodyManipulator.h
fixed a few whitespace issues in template arguments.
updated urdf loader by restoring commented out error / debug messages (now it might be too much spew).

note: default URDF loader (imported from ROS) set limits to zero and errored out if there were no effort/velocity limits.  but i decided that i had to respect the urdf spec in doc/drakeURDF.xsd, since that's what we've been using.  not real happy about that one.

Oh, and CONGRATS PATS!!

### CODE REVIEW

File: util/makeROSCompatibleURDF.m [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  end  
+end
+
+xmlwrite(output_urdf_filename,urdf);
+fprintf('Output written to %s\n',output_urdf_filename);
+
+end
+
+
+function param_db = parseParameter(param_db,node)
```
Comment (avalenzu) [inline util/makeROSCompatibleURDF.m]: Is there a way we could avoid duplicating this function between RBM and here?

File: systems/plants/urdf_interface/joint.cpp [hunk 2]
```diff
@@ -197,7 +208,7 @@ bool JointSafety::initXml(TiXmlElement* config)
   const char* soft_lower_limit_str = config->Attribute("soft_lower_limit");
   if (soft_lower_limit_str == NULL)
   {
-    //ROS_DEBUG("joint safety: no soft_lower_limit, using default value");
+    ROS_DEBUG("joint safety: no soft_lower_limit, using default value");
```
Comment (avalenzu) [inline systems/plants/urdf_interface/joint.cpp]: For some of the other attributes the debug message lists what the default value is. Do we want to do that throughout?

**Comment by avalenzu** [comment]
Left some minor notes inline. I'd say it's ok to merge as is though.

File: util/makeROSCompatibleURDF.m (hunk 1 above)
Comment (RussTedrake) [inline util/makeROSCompatibleURDF.m]: unfortunately, it's behavior is a sufficiently different that I think duplication+modification is probably the right way to go.

**Pull request merged**